### ISSUE TITLE
Building nginx with custom changes for debugging

### ISSUE THREAD

**Issue by workhorse2020** [nginxinc/kubernetes-ingress#2104]
Hi All,

  Was working on creating a clone and building with custom changes and I did face issue when initiating a build. Please let me know if this building is only allowed for Nginx plus ? .

```
make debian-image PREFIX=myregistry.example.com/nginx-ingress TARGET=container
Docker version 18.06.1-ce, build e68fc7a
docker build --build-arg IC_VERSION=v1.12.0-SNAPSHOT-1af886a --build-arg GIT_COMMIT=1af886a2fffdf3b7eb88028383474f1ec2d43e3f --build-arg DATE=2021-10-18T05:48:04Z --target container -f build/Dockerfile -t myregistry.example.com/nginx-ingress:1.12.0-SNAPSHOT-1af886a . --build-arg BUILD_OS=debian
buildkit not supported by daemon
make: *** [debian-image] Error 1
```

rgds
Kamal

**Comment by lucacome** [comment]
Hi @workhorse2020,

we’re using BuildKit to build the images and it’s not available in Docker 18.06 (the one you’re using). You would need to update to at least version 18.09.

**Comment by bkannadassan** [comment]
@lucacome  Thanks it worked fine..
